opensearch-dashboards-2.yaml

```diff
@@ -63,6 +63,10 @@ pipeline:
       cherry-picks: |
         main/5e19749ec40230316ba2688c38e5c62f74ddb71d: CVE-2024-37890
 
+  - uses: patch
+    with:
+      patches: cve-2024-21538.patch
```

Comment:
This is good, but I am still not convinced with adding patches which aren't cleaned it up :/.

Reply:
yeah though it's no different from what's happening today when someone adds any node CVE remediation

cleaning these up definitely needs to be automated too.